examples/assemblyscript/assemblyscript_test.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/tetratelabs/wazero/internal/testing/maintester"
+	"github.com/tetratelabs/wazero/internal/testing/require"
+)
+
+// Test_main ensures the following will work:
+//
+// go run assemblyscript.go 7
+func Test_main(t *testing.T) {
+	if runtime.GOARCH == "riscv64" {
+		t.Skip("Uses bulk memory operations which are not implemented on riscv64 yet.")
```

Comment:
looking at the ci error, this is not due to bulk memory thingy (as the runtime passes all spec tests), but due the usage of AOT compiler on riscv...